src/config/agent_config.go

```
@@ -10,8 +10,9 @@ import (
 type AgentConfig struct {
 	DryRunConfig
 	ConnectivityConfig
-	IntervalSecs int
-	HostID       string
+	IntervalSecs          int
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)